Core/Graphics/Include/Android/Babylon/GraphicsPlatform.h

```diff
@@ -1,15 +1,23 @@
  #pragma once
 
 #include <android/native_window.h>
+#include <EGL/egl.h>
 
 namespace Babylon
 {
     using WindowType = ANativeWindow*;
+    using ContextType = EGLContext;
 
     struct GraphicsConfiguration
     {
         WindowType WindowPtr;
         size_t Width;
         size_t Height;
     };
+
+    struct ContextConfiguration
```

Comment:
This is minor, but I feel like `GraphicsConfiguration` and `ContextConfiguration` are not good names. It seems like it should be `Window` vs `Context` and not `Graphics` vs. `Context`. Can we rename `GraphicsConfiguration` to `WindowConfiguration` maybe?

Reply:
I originally intended for Graphics configuration to be _the_ config struct, do we want to have multiple structs for different uses?

Reply:
I have no strong feelings for 1 function with more members in the struct or 2 more explicit functions. Maybe 2 functions make the contract more clear and it simplifies checking logic behind (having both window and context set for example).

Reply:
I think two makes the contract a lot more self explanatory since you have to pick one or the other.

Reply:
Is the expectation then that the chosen struct type must be comprehensively populated? If so, unless I misunderstand, that seems like it goes against one of the primary arguments for using structs in the first place, which (as I recall) was that using such a config struct would make it easier to update the available contracts by simply extending the struct and allowing consumers to leave new fields empty if they didn't want to use them. If we're going to have a different struct for every use case, is there an advantage to having structs at all versus just having overloads?